### PR TITLE
BBBSWA-26 merge with main

### PR DESCRIPTION
**Pull request by rulanlu**

### CODE REVIEW

**Review by ckthiessen — APPROVED** [review]
Some changes requested

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-firebaseCredentials.json
```
Comment (ckthiessen) [inline .gitignore:1]: Why did you remove this from gitignore?

File: frontEnd/src/components/Header.vue [hunk 2]
```diff
@@ -30,7 +66,7 @@
             rounded 
             elevation="0"
           >
-            Steve Buscemi
```
Comment (ckthiessen) [inline frontEnd/src/components/Header.vue:33]: Sad to see Steve go :(

File: frontEnd/src/components/Header.vue [hunk 3]
```diff
@@ -77,9 +114,45 @@
...
             menu: false,
+            notifications:[],
+            name: '',
         }),
+
+        created(){
+          this.getUser(); //call when instance is new
+        },
+        methods: {
+          async getUser(){
```
Comment (ckthiessen) [inline frontEnd/src/components/Header.vue]: Why use wrapper function for getUserByID? getUserByID is already a wrapper function for the Axios call so I would delete getUser() and just use getUserById()

**Review by ckthiessen — CHANGES_REQUESTED** [review]

File: frontEnd/src/components/Header.vue (hunk 2 above)
Comment (rulanlu) [reply frontEnd/src/components/Header.vue:33]: RIP Steve :(

File: .gitignore (hunk 1 above)
Comment (rulanlu) [reply .gitignore:1]: It was already in there at the bottom of .gitignore from when I pulled! Just didn't see it

File: frontEnd/src/components/Header.vue (hunk 3 above)
Comment (rulanlu) [reply frontEnd/src/components/Header.vue]: Ah I just copied the formatting from HelloWorld.vue originally to match the style, forgot to change it after though oops

Comment (rulanlu) [reply frontEnd/src/components/Header.vue]: I'll do that now

File: .gitignore (hunk 1 above)
Comment (ckthiessen) [reply .gitignore:1]: Okay sounds good!
